src/Microsoft.Health.Fhir.Core/Features/Conformance/Models/ListedResourceComponent.cs

```diff
@@ -21,7 +21,10 @@ public ListedResourceComponent(CoreFeatureConfiguration configuration)
             SearchRevInclude = new HashSet<string>(StringComparer.Ordinal);
             SearchInclude = new HashSet<string>(StringComparer.Ordinal);
             ReferencePolicy = new HashSet<string>(StringComparer.Ordinal);
-            Versioning = new DefaultOptionHashSet<string>(configuration.Versioning.Default, StringComparer.Ordinal);
+
+            var versioningPolicy = configuration.Versioning.Default;
+
+            Versioning = new DefaultOptionHashSet<string>(versioningPolicy, StringComparer.Ordinal);
```

Comment:
With your new changes in the builder, can this now be simplified, Configuration probably doesn't need to be passed in anymore

```suggestion
            Versioning = new DefaultOptionHashSet<string>(ResourceVersionPolicy.Versioned, StringComparer.Ordinal);
```

Reply:
This has now been simplified.